api/httpclient.go

```diff
@@ -19,17 +19,22 @@ type headerTransport struct {
 // NewHTTPClientWithHeaders returns a *http.Client that attaches a defined set of Headers to all requests.
 // If specified, the client will also trust the CA certificate specified in the client configuration.
 func (c *Client) newHTTPClientWithHeaders(headers map[string]string) *http.Client {
+	dialContext := c.config.DialContext
+	if dialContext == nil {
+		dialContext = (&net.Dialer{
+			Timeout:   30 * time.Second,
+			KeepAlive: 30 * time.Second,
+			DualStack: true,
```

Comment:
Looks like this field was deprecated when looking up the docs for it https://golang.org/pkg/net/#Dialer:
```
    // DualStack previously enabled RFC 6555 Fast Fallback
    // support, also known as "Happy Eyeballs", in which IPv4 is
    // tried soon if IPv6 appears to be misconfigured and
    // hanging.
    //
    // Deprecated: Fast Fallback is enabled by default. To
    // disable, set FallbackDelay to a negative value.
```

Reply:
Yea. But its defined in `http.DefaultTransport` which is what is mirrored here. Also just moved this up from a few lines down.